### ISSUE TITLE
[BetterQuesting] line drawing issues in quest book

### ISSUE THREAD

**Issue by combusterf** [GTNewHorizons/NewHorizons#4233]
#### Which modpack version are you using?
2.0.7.0

#
#### If in multiplayer; On which server does this happen? 
Delta (issue did not happen before the update)

#
#### What did you try to do, and what did you expect to happen?
Opening up the questbook and view a quest tier map

#
#### What happend instead? (Attach screenshots if needed)
Some lines are aliased on the edge of the viewport, some cross into other UI components. Dragging the viewport changes the locations of lines and their colours. Scrolling to the same position within the view results in the same render errors:
 
![image](https://user-images.githubusercontent.com/8459776/54089494-a6c3fd00-4369-11e9-8501-1688ef019dc5.png)

#
#### What do you suggest instead/what changes do you propose?
Since this looks like a regression bug, find the responsible change and fix/revert it

**Comment by richardhendricks** [comment]
Doesn't seem to happen to me. Do other Delta players see this?  Might need to redo default load on your character in Delta.

**Comment by boubou19** [comment]
i'll see if it happens to me too

**Comment by Dream-Master** [comment]
I never had this too.

**Comment by boubou19** [comment]
i don't have this issue on delta

**Comment by richardhendricks** [comment]
@combusterf Recommend doing a test instance and see if you still have the issue.  It is possible we would need the exact quest progress you are at to duplicate.  Are you on a public server or private server/ssp?

**Comment by combusterf** [comment]
I've been poking at it a bit more, and I've noticed that I can predictably make the left side light up by scrolling any perfectly vertical line off the left side of the viewport - at which point the entire left boundary takes over the colour (including blinking when appropriate).

I have not yet figured a predictable method for the top boundary.

- I've rotated through the GUI scale config option and could reproduce it in all modes. Screenshot is in Large scale
- Claiming (all) rewards on a page does not fix the issue
- The issue is not limited to the LV page; the method above can be executed on the majority of pages.
- Reproducable both in SMP (Delta) and a fresh SSP game - once the first 16 dirt is collected there is a vertical line in the questbook to use as per the above.
